apps/torus-wallet/src/app/components/sidebar-links.tsx

```
@@ -10,84 +10,109 @@ import {
-              {isActive("/staking") && <Check size={16} />}
-            </Link>
-          </Button>
-          <Button
-            asChild
-            variant="ghost"
-            className={`w-full justify-between gap-4 border-none px-3 text-base`}
-          >
-            <Link href={bridgeLink}>Bridge</Link>
-          </Button>
+          {NAVIGATION_LINKS.map(navLinkProps => (
+            <NavButton {...navLinkProps} />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)